source/elements/oneDAL/source/data_management/index.rst

```diff
@@ -5,12 +5,140 @@
 Data management
 ===============
 
-The Data Management component includes classes and utilities for data acquisition, initial preprocessing
-and normalization, for data conversion into numeric formats (performed by one of supported Data Sources)
-and for model representation.
+Effective data management is among key constituents of the performance of a data
```

Comment:
I'd not use 'effective', 'among key' etc. Those are just specs which describe the component of the library. Pls, use style and language of the text block removed